pybossa/task_creator_helper.py

```diff
@@ -64,7 +64,6 @@ def set_gold_answers(task, gold_answers):
     if encrypted():
         url = upload_files_priv(task, task.project_id, gold_answers, TASK_PRIVATE_GOLD_ANSWER_FILE_NAME)['externalUrl']
         gold_answers = dict([(TASK_GOLD_ANSWER_URL_KEY, url)])
```

Comment:
I think setting gold task should not be setting task expiration as both operations are independent and hence task expiration not to be modified with setting gold task.

@kbecker42 , @XiChenn , @n00rsy thoughts?

Reply:
Looks like a business side decision to me.